src/RhinoInside.Revit.GH/Components/Element/SpatialElement/AddRoomSeparatorLine.cs

```diff
@@ -107,12 +108,14 @@ protected override void TrySolveInstance(IGH_DataAccess DA)
       );
     }
 
-    bool Reuse(ARDB.ModelCurve roomSeparator, ARDB.Document doc, Curve curve, ARDB.View view)
+    bool Reuse(ARDB.ModelCurve roomSeparator, Curve curve, ARDB.View view)
     {
       if (roomSeparator is null) return false;
       if (roomSeparator.OwnerViewId != view.Id) return false;
       var projectedCurve = Curve.ProjectToPlane(curve, view.SketchPlane.GetPlane().ToPlane());
       if (!projectedCurve.ToCurve().IsAlmostEqualTo(roomSeparator.GeometryCurve)) return false;
+
```

Comment:
returning `false` here prevents the curve is updated.

Reply:
that's because when the curve is different we return false in Reuse and go to Create method, isn't it?

Reply:
Sure, but the goal of `Reuse` is update the element with the new curve.
Reuse should return `false` only when is unable to update the element.

Reply:
Makes sense! Done!